src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb

```diff
@@ -50,6 +50,14 @@ Public Module BuildDevDivInsertionFiles
             "codeAnalysisService.servicehub.service.json",
             "remoteHostService.servicehub.service.json",
             "serviceHubSnapshotService.servicehub.service.json",
+            "Microsoft.Build.Conversion.Core.dll",
+            "Microsoft.Build.dll",
+            "Microsoft.Build.Engine.dll",
+            "Microsoft.Build.Framework.dll",
+            "Microsoft.Build.Tasks.Core.dll",
+            "Microsoft.Build.Utilities.Core.dll",
+            "Microsoft.VisualStudio.Threading.dll",
+            "Microsoft.VisualStudio.Validation.dll",
```

Comment:
already talked to Kevin, not sure why we have put MSBuildWorkspace in Workspace.Desktop.dll. which cause any one who want to use Worskapce.Desktop.dll to use those service to now stuck with that dependencies.